test/crossfilter-test.js

```diff
@@ -3520,7 +3520,32 @@ suite.addBatch({
         }
       },
     },
+    "filter the 32nd dimension": function() {
+      var dataSet = [];
 
+      var itemBuilder = (fieldCount, value) => {
+        var item = {};
+        for (var i = 0; i < fieldCount; i++) {
+          item['f' + (i + 1)] = value;
+        }
+        return item;
+      }
+
+      dataSet.push(itemBuilder(34, 'a'));
+      dataSet.push(itemBuilder(34, 'b'));
+      var data = crossfilter(dataSet);
+
+      var dimensions = Object.keys(dataSet[0]).map(key => data.dimension(d => d[key]));
+      var groups = dimensions.map(d => d.group());
+      dimensions[31].filterExact('a');
+      // correct group value
+      const correctGroupValue = groups.map(g => g.all()[1].value);
```

Comment:
```suggestion
      const correctGroupValue = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0];
      assert.deepEqual(groups.map(g => g.all()[1].value), correctGroupValue);
```
This way future readers will know what is expected without adding `console.log` or otherwise running the code to find out the values. It's also not bad to assert the expected value before we hit the bug as well as after (even though we know it will succeed here).